Stripe/PublicHeaders/STPUserInformation.h

```diff
@@ -38,6 +42,18 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property(nonatomic, copy, nullable)NSString *idealBank;
 
+/**
+ *  Metadata associated with your user. This will be attached to any Source
+ *  objects created by `STPPaymentContext`, `STPAddSourceViewController`, etc.
+ *  You should consider storing any order information (e.g., order number) here.
+ *  For payment methods that require additional user action, your backend will
+ *  need to listen to the `source.chargeable` webhook to create a charge request.
+ *  You can retrieve this metadata from the webhook event, and use it to fulfill
+ *  your customer's order.
+ *  https://stripe.com/docs/sources#best-practices
+ */
+@property(nonatomic, copy, nullable)NSDictionary *metadata;
```

Comment:
Should add a comment about what types of objects are valid here. Maybe even do `NSDictionary<NSString *, id> *` at minimum, or enforce all strings only.